test/unit/preact/test-base-element-runtime.js

```diff
@@ -79,6 +79,19 @@ describes.realWin('PreactBaseElement', {amp: true}, (env) => {
     });
   }
 
+  it('preact ref passing vnode smoke test', () => {
```

Comment:
Incredibly smart test for guaranteeing we clean up the hack. A+ I'm very impressed